src/content/plots/d3/ContourD3.tsx

```diff
@@ -85,38 +91,41 @@ const create = (el, layout, x: number[], y: number[]) => {
 
     // *//
 
-    const contour_bins = 50
-    const bins = rectbin(x, y, contour_bins)
+    // Calculate and smooth bins
+    const bins = rectbin(x, y, config.bins)
+    d3.blur2({ data: bins, width: config.bins }, config.blur_radius)
+
+    // Calculate the contours
     const contours = d3
         .contours()
-        .size([contour_bins, contour_bins])
+        .size([config.bins, config.bins])
         .thresholds((v: number[]) => {
             const v_sort = d3.sort(v)
             const v_cum = d3.cumsum(v_sort)
             const thresholds = sigmas.map(v => (1 - v) * v_cum[v_cum.length - 1])
             return thresholds.map(v => v_sort[v_cum.findIndex(v_c => v_c > v)])
         })(bins)
 
-    const contour_x_scale = layout.width / contour_bins
-    const contour_y_scale = layout.height / contour_bins
+    // Transform contours to the correct size for the svg
+    const contour_x_scale = layout.width / config.bins
```

Comment:
Usually JS/TS is written using camel case. Is there a reason snake case was used instead?

Reply:
I think it's fine to leave variables in snake case